<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>281</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius: 10px;
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLineEdit" name="username_input">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#27a102;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_input">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#27a102;
border-radius: 10px;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_bt">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#55ff00</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="invalid">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>241</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffffff;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>login_bt</zorder>
    <zorder>invalid</zorder>
    <zorder>password_input</zorder>
    <zorder>username_input</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="username_display">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffffff;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#27a102;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>730</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffaa00;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="expiry_date">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color:white;</string>
     </property>
     <property name="text">
      <string>Expiry/Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffaa00;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>download (1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_account">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Segoe UI&quot;;
color:white;</string>
     </property>
     <property name="text">
      <string>Card Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="available_balance_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 6pt &quot;Segoe UI&quot;;
color:white;</string>
     </property>
     <property name="text">
      <string>Available Balance</string>
     </property>
    </widget>
    <widget class="QLabel" name="bank_display">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>â‚±:100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Transaction History</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="transfer_bt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Transfer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Recieve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Pay Bills</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>username_display</zorder>
    <zorder>label_9</zorder>
    <zorder>label</zorder>
    <zorder>expiry_date</zorder>
    <zorder>label_2</zorder>
    <zorder>display_account</zorder>
    <zorder>available_balance_label</zorder>
    <zorder>bank_display</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>transfer_bt</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffffff;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="transfer_input_id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="transfer_price_input">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
    </widget>
    <widget class="QLabel" name="balance_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#27a102;
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="home_bt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QLabel" name="invalid_label_2">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>30</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>201</width>
       <height>181</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="normal_page">
      <widget class="QPushButton" name="transfer_money_bt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 127);</string>
       </property>
       <property name="text">
        <string>Transfer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="transfer_page"/>
     <widget class="QWidget" name="are_you_sure_page">
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Are you sure you send this to?</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="transfer_money_bt_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 127);</string>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="transfer_money_bt_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 127);</string>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="transfer_money_bt_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>transfer_price_input</zorder>
    <zorder>transfer_input_id</zorder>
    <zorder>balance_label</zorder>
    <zorder>home_bt</zorder>
    <zorder>invalid_label_2</zorder>
    <zorder>stackedWidget_2</zorder>
    <zorder>transfer_money_bt_2</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Nutrional_app - Copy/images (1).jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
